script/hassfest/config_schema.py

```diff
@@ -20,7 +20,7 @@ def _has_assignment(module: ast.Module, name: str) -> bool:
             continue
         if type(item) == ast.Assign:
             for target in item.targets:
-                if target.id == name:
+                if type(target) == ast.Name and target.id == name:
```

Comment:
```suggestion
                if getattr(target, "id", None) == name:
```

`getattr` might be simpler.

Reply:
Good idea